include/openvswitch/ofp-msgs.h

```diff
@@ -190,6 +190,10 @@ enum ofpraw {
     OFPRAW_NXT_LOAD_FILTER_PROG,
     /* NXT 1.0+ (32): struct ol_update_map, uint8_t[]. */
     OFPRAW_NXT_UPDATE_MAP,
+    /* NXT 1.0+ (33): struct ol_dump_map_request, uint8_t[]. */
```

Comment:
I don't think `OFPRAW_NXT_DUMP_MAP_REQUEST` needs `uint8_t[]` since it's a fixed-size message.
```suggestion
    /* NXT 1.0+ (33): struct ol_dump_map_request. */
```